BraveRewardsUI/RewardsDataSource.swift

```diff
@@ -13,6 +13,6 @@ public protocol RewardsDataSource: AnyObject {
   /// Download or retrieve a cached version of the favicon given a URL. This should also return the
   /// default letter/color favicon if a website has no official favicon.
   ///
-  /// Execute completionBlock with an image (or nil) when its available
-  func retrieveFavicon(with url: URL, completion: @escaping (UIImage?) -> Void)
+  /// Execute completionBlock with an image and background color (or nil) when its available
+  func retrieveFavicon(with url: URL, completion: @escaping (UIImage?, UIColor?) -> Void)
```

Comment:
Is there ever a point where we'd send non-nil image and nil color (or vice-versa)? 

If not, we should probably just package this up into a struct, like `Favicon` or an optional tuple.